src/server/scripts/Northrend/CrusadersColiseum/TrialOfTheChampion/boss_argent_challenge.cpp

```diff
@@ -17,64 +17,118 @@
 
 /* ScriptData
 SDName: Argent Challenge Encounter.
-SD%Complete: 50 %
-SDComment: AI for Argent Soldiers are not implemented. AI from bosses need more improvements.
+SD%Complete: 100%
```

Comment:
We may want to get rid of the ScriptData (SD) comment section at a later date, when we are reasonably certain we have replaced the MaNGOS code remnants in the script. Comments are welcome on this part, since I am not certain about the details myself.